src/components/components/HeroSection/HeroSection.js

```diff
@@ -0,0 +1,35 @@
+import React from "react";
+import "./HeroSection.scss";
+import HeroImage from "./assets/hero-image.svg";
+
+function HeroSection() {
+	return (
+		<div className="HeroSection">
+			<div className="HeroSection__info">
+				<h1 className="HeroSection__title">
+					Uniting <span className="HeroSection__title--circled">skill</span>{" "}
+					with opportunity
+				</h1>
+				<p className="HeroSection__text">
+					SkillChain connects job seekers with top employers to find the perfect
+					match for job vacancies.
+				</p>
+				<a
+					href="https://www.google.de/"
+					target="_blank"
+					rel="noopener noreferrer"
+				>
+					<button className="HeroSection__button">View Jobs</button>
```

Comment:
A button inside of an a-tag? Does this make sense?

Reply:
no, you right. I shall change it